src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -8,27 +8,127 @@
 ### Normal Counter
 
 ```
-<NcCounterBubble>314+</NcCounterBubble>
+<NcCounterBubble count=314 />
 ```
 
-### Outlined Counter (e.g team mentions)
+### Limited
 
 ```
-<NcCounterBubble type="outlined">314+</NcCounterBubble>
+<template>
+	<table>
+		<tr>
+			<th>Limit</th>
+			<th>Count</th>
+			<th>NcCounterBubble</th>
+		</tr>
+		<tr>
+			<td>999 (default)</td>
+			<td>1000</td>
+			<td>
+				<NcCounterBubble count="1000" />
+			</td>
+		</tr>
+		<tr>
+			<td>12</td>
+			<td>9</td>
+			<td>
+				<NcCounterBubble count="12" limit="9" />
+			</td>
+		</tr>
+		<tr>
+			<td>0 (disabled)</td>
+			<td>1000</td>
+			<td>
+				<NcCounterBubble count="1000" limit="0" />
+			</td>
+		</tr>
+	</table>
+</template>
+
+<style scoped>
+table {
+	border-collapse: collapse;
+}
+th,
+td {
+	border: 1px solid var(--color-border);
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
+}
+
+th {
+	color: var(--color-text-maxcontrast);
+}
+</style>
 ```
 
-### Highlighted Counter (e.g direct mentions)
+### Style type
 
 ```
-<NcCounterBubble type="highlighted">314+</NcCounterBubble>
+<template>
+	<table>
+		<tr>
+			<th>type</th>
+			<th>NcCounterBubble</th>
+			<th>Usage example</th>
+		</tr>
+		<tr>
+			<td>'' (default)</td>
+			<td>
+				<NcCounterBubble count="314" />
+			</td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>outlined</td>
+			<td><NcCounterBubble type="outlined" count="314" /></td>
+			<td>Team mentions</td>
+		</tr>
+		<tr>
+			<td>highlighted</td>
+			<td>
+				<NcCounterBubble type="highlighted" count="314" />
+			</td>
+			<td>Direct mentions</td>
+		</tr>
+	</table>
+</template>
+
+<style scoped>
+table {
+	border-collapse: collapse;
+}
+th,
+td {
+	border: 1px solid var(--color-border);
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
+}
+
+th {
+	color: var(--color-text-maxcontrast);
+}
+</style>
 ```
 
+### Custom content
+
+Deprecated. Use the `count` prop instead.
+
+```
+<NcCounterBubble>314+</NcCounterBubble>
+````
+
 </docs>
 
 <template>
 	<div :class="counterClassObject"
 		class="counter-bubble__counter">
-		<slot />
+		<!--
+			@slot The content with the count number.
```

Comment:
```suggestion
			@slot (deprecated) The content with the count number.
```

Because styleguide output is not that clear I think

Reply:
It has strikethrough

Reply:
> It has strikethrough

I saw this, but it was not clear that this is deprecated or what there is a strike through